_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -958,7 +1152,68 @@ link:https://docs.aws.amazon.com/general/latest/gr/aws_tasks-that-require-root.h
 the use of root].  Fortunately, most of these activities are rare, so usage of the root account can be kept to
 a minimum.
 
-Next, continue with the <<configure_logging>> section.
+[[configure_storage]]
+=== Configure storage
+
+[[s3_buckets_deployment]]
+==== S3 Buckets
+To make sure your S3 buckets are compliant with the benchmark, use the
+link:https://github.com/gruntwork-io/terraform-aws-security/tree/master/modules/private-s3-bucket[`private-s3-bucket` module]
+to create and manage all of your S3 buckets. This module blocks public access and enforces encryption by default. Note
+that all Gruntwork modules that create S3 buckets use this module under the hood.
+
+To see a usage example of the `private-s3-bucket` module, refer the
+link:the https://github.com/gruntwork-io/terraform-aws-service-catalog/blob/master/modules/data-stores/s3-bucket/[`s3-bucket` service]
+in the Gruntwork Service Catalog.
+
+[[encrypt_ebs_volumes]]
+==== Encrypt EBS volumes
+
+Use the link:https://github.com/gruntwork-io/terraform-aws-security/tree/master/modules/ebs-encryption-multi-region[`ebs-encryption-multi-region` module]
+to configure AWS EBS encryption in all enabled regions of an AWS Account, as per the Benchmark recommendation.
+
+If your using Terragrunt, then you can invoke the standalone module directly by following the pattern outlined in
+<<wrapper_code_terragrunt>> and creating a `terragrunt.hcl` file with the following contents:
+
+.infrastructure-live/<your account>/_global/ebs-encryption/terragrunt.hcl
+[source,hcl]
+----
+terraform {
+  # Make sure to replace <VERSION> in this URL with the latest terraform-aws-security release
+  source = "git::ssh://git@github.com/gruntwork-io/terraform-aws-security.git//modules/ebs-encryption-multi-region?ref=<VERSION>"
+}
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+inputs = {
+  enable_encryption = true
+  seed_region       = "us-east-2"
+}
+----
+
+[source,hcl]
+----
+# ----------------------------------------------------------------------------------------------------------------------
+# ELASTIC BLOCK STORAGE (EBS) ENCRYPTION DEFAULTS
```

Comment:
What's this for? Seems like a left over example?

Reply:
probs - will review and look how it looks now!

Reply:
removed the whole block - it seems like we had what we needed in there.